CacheManager.StackExchange.Redis.SelfCleaning.Examples/Scenarios/SimpleSingleExpiredItemScenario.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using CacheManager.Core;
+using CacheManager.Core.Internal;
+
+namespace CacheManager.StackExchange.Redis.SelfCleaning.Examples.Scenarios
+{
+    public class SimpleSingleExpiredItemScenario : BaseSingleScenario<int>
+    {
+        private ICacheManager<int> _cacheManager;
+
+        private CacheItem<int> _cacheItem;
+        private List<CacheItemRemovedEventArgs> _actualRemovedCacheItems;
```

Comment:
1. Not critical, but maybe change this to ICollection<> instead (just for the sake of conventions and looks)
2. You can make this field `readonly`

Reply:
f & f